getting-started/proc-configuring-github-app.adoc

```diff
@@ -0,0 +1,24 @@
+[id="proc-configuring-github-app"]
+
+= Configuring a GitHub App
```

Comment:
We elaborate on the heading for the user's clarity. Where the user needs to configure the GitHub app?